src/susceptibility.cpp

```diff
@@ -179,7 +179,7 @@ void lorentzian_susceptibility::update_P
 
   if (!no_omega_0_denominator && gamma >= 0
       && lorentzian_unstable(omega_0, gamma, dt))
-    abort("Lorentzian pole at too high a frequency %g for stability with dt = %g: reduce the Courant factor, increase the resolution, or use a different dielectric model\n", omega_0, dt);
+    master_printf("WARNING: simulation is potentially unstable as the Lorentzian pole is at too high a frequency %g for stability with dt = %g. In the event the fields diverge: reduce the Courant factor, increase the resolution, or use a different dielectric model\n", omega_0, dt);
```

Comment:
I don't think we want to print out a warning on every timestep.

Reply:
Maybe we should just omit the warning entirely.  We don't warn about other potential instabilities.